### ISSUE TITLE
pip install mkdocs-markdownextradata-plugin results in an error

### ISSUE THREAD

**Issue by shaniR** [rosscdh/mkdocs-markdownextradata-plugin#11]
Installing results in an error.
pip install mkdocs-markdownextradata-plugin
Collecting mkdocs-markdownextradata-plugin
  Using cached https://files.pythonhosted.org/packages/71/2d/64f8d3871a69830b07fe184f94b050244b1cd0ff06fa53e25de4997a7dcc/mkdocs-markdownextradata-plugin-0.1.0.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-2ltd1K/mkdocs-markdownextradata-plugin/setup.py", line 27, in <module>
        long_description=read('README.md'),
      File "/tmp/pip-build-2ltd1K/mkdocs-markdownextradata-plugin/setup.py", line 8, in read
        return Path(os.path.join(os.path.dirname(__file__), fname)).read_text()
    AttributeError: 'PosixPath' object has no attribute 'read_text'

**Comment by rosscdh** [comment]
Hi there,

are you using python 2? or python 3?

**Comment by rosscdh** [comment]
Made it p2.7 friendly again

as of version 0.1.1 https://pypi.org/project/mkdocs-markdownextradata-plugin/#history

**Comment by shaniR** [comment]
It didn't work for python3 as well. now it works for python 2. Will check with python 3 and update if the error still is found. Thank you so much.

**Comment by rosscdh** [comment]
am using/testing on python 3.7 so id be surprised if it doesnt. Thanks for taking the time to lodge the ticket please let me know how it goes

**Comment by rosscdh** [comment]
https://github.com/aio-libs/aiohttp/issues/3977 as per 

```
pathlib.Path.read_text() should available under Python 3.5+. If you don't have it, your Python installation is probably broken.
```
